src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
It's recommended to not use public static fields for storing data. Instead, you could create a private field and provide access to it via public methods. This allows for better control over how the data is accessed and modified.